pkg/control/service_control.go

```diff
@@ -123,7 +125,9 @@ func (r RealServiceControl) DeleteService(namespace, serviceID string, object ru
 	if err != nil {
 		return fmt.Errorf("object does not have ObjectMeta, %v", err)
 	}
-	glog.V(2).Infof("Controller %v deleting service %v/%v", accessor.GetName(), namespace, serviceID)
+	if log.GetLevel() >= log.ErrorLevel {
```

Comment:
I think we could just use log.Infof without checking the level. Do you think it is necessary?

Reply:
no need to check level, fix it right now